travis/licenses.sh

```diff
@@ -1,3 +1,4 @@
+set -ex
```

Comment:
Was the `x` intentional?

Reply:
This is for consistency with some of the other Travis scripts (and possibly helpfui for debugging)